source/operational-readiness-checklist.txt

```diff
@@ -0,0 +1,318 @@
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: onecol
+
+This checklist is designed to help you prepare your environment and team
+for a successful deployment and operation of |service-fullname|.
+Use this checklist to track your progress. For example, you can print it
+and check off each item as you complete the tasks.
+
+Consult the official |service-fullname| documentation for detailed guidance
+on each of these aspects.
+
+Account and Organization Setup
+-------------------------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 5 55
+
+   * - Check
+     - Action
+
+   * - :icon-fa4:`square-o`
+     - Create a |service-fullname| account, set up your |service| organizations
+       according to your internal structure, and configure a root user
+       with appropriate access. To get recommendations and learn more about
+       this topic, see :ref:`arch-center-hierarchy`.
+
+   * - :icon-fa4:`square-o`
+     - Set up projects based on your environment and application needs.
+       Establish multi-tenancy strategy by setting up production and non-production
+       projects, at a minimum. To get recommendations and learn more about
+       this topic, see :ref:`arch-center-orgs-projects-clusters-recs`.
+
+   * - :icon-fa4:`square-o`
+     - Consider cross-prganization billing, if applicable. To get recommendations
+       and learn more about this topic, see :ref:`arch-center-billing-data`.
+
+Network and Security Configuration
+-----------------------------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 5 55
+
+   * - Check
+     - Action
+
+   * - :icon-fa4:`square-o`
+     - Select cloud providers and regions for your |service| clusters.
+       Consider data sovereignty requirements and latency. To get
+       recommendations and learn more about this topic, see
+       :ref:`arch-center-paradigms`.
+
+   * - :icon-fa4:`square-o`
+     - Configure network security based on your organization's needs.
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-auth`.
+
+   * - :icon-fa4:`square-o`
+     - Choose a network connectivity method:
+       
+       - :ref:`VPC Peering <arch-center-vpc-vnet-peering>` for setting up
+         private and secure traffic routing within your network boundaries.
+       - :ref:`Private Endpoints <arch-center-private-endpoints>`
+         (AWS PrivateLink, Azure Private Link, or
+         Google Cloud Private Service Connect) for a one-way private
+         connection from your VPC to |service|.
+         For sharded multi-region clusters, enable private endpoints in each region. To learn more, see :ref:`Recommendations for Multi-Region
+         Deployments <arch-center-multi-region-recs-private-endpoints>`.
+       - Public :ref:`IP Access Lists <arch-center-ip-access-list>`
+         to restrict inbound connections to specific IP addresses or CIDR
+         blocks. Consider IP Allow lists as an alternative to Private Link if necessary.
```

Comment:
we should make this slightly more generic so it can apply for all three cloud providers
```suggestion
         blocks. Consider IP Allow lists as an alternative to private endpoints if necessary.
```